endoreg_db/models/other/emission/emission_factor.py

```
@@ -1,5 +1,13 @@
 from django.db import models
-from typing import List
+from typing import TYPE_CHECKING, List
+
+from endoreg_db.models.administration.product import reference_product
```

Ruff (0.8.2)

`endoreg_db.models.administration.product.reference_product` imported but unused

Remove unused import: `endoreg_db.models.administration.product.reference_product`

(F401)